public/javascripts/PullInventory.js

```diff
@@ -18,10 +18,18 @@ var pullInventory = {
     init: function(){
 
         //grab and save navigation object arguments
-        this.navigationArgs.productID = window.args.ProductID;
```

Comment:
This can be shortened to:
this.navigationArgs.productID = window.args.ProductID || window.state.ProductID;
this.navigationArgs.productName = window.args.ProductName || window.state.ProductName;
this.navigationArgs.previousPage = window.args.PreviousPage || window.state.PreviousPage;
this.navigationArgs.totalQuantity = window.args.TotalQuantity || window.state.TotalQuantity;